projects/vs/psa-boot-test/psa-boot-testdata.h

```diff
@@ -4,7 +4,7 @@
 
 #include <stdint.h>
 
-static const uint8_t td_valid_package[] = {
+static const uint8_t td_package_valid[] = {
```

Comment:
Why did you declare variables in header file?
Header files shouldn't include variable declarations.

Reply:
Actually, I have created this file. You are right, just this file is not an interface that hides data, instead, it keeps public test data which is tested by the test code. It is public information anyone can access. 
The idea was that separate the test data and the test code.

Reply:
It's ok if only included by one file but still not good practice. I think making this proper C way is better. Move all of these declarations to psa-boot-testdata.c file and extern them in this header file.

Reply:
I don't see a problem, we could keep the test data inside the test source file to access all attributes(content/sizeof) of the test data, "extern" may mask some of them. 

Some C Test frameworks do the same thing; for example PSA Architecture Test  (please see the test_data.h)
https://github.com/ARM-software/psa-arch-tests/tree/master/api-tests/dev_apis/crypto/test_c002

It is not the bootloader implementation itself, it is just a private resource for the test code.